rust/src/config.rs

```diff
@@ -9,21 +9,40 @@ const MAINNET_ELECTRUM: &str = "ssl://blockstream.info:700";
 const TESTNET_ELECTRUM: &str = "ssl://blockstream.info:993";
 const REGTEST_ELECTRUM: &str = "tcp://localhost:50000";
 
-static REGTEST_MAKER_IP: &str = "127.0.0.1";
-static REGTEST_MAKER_PORT_HTTP: u64 = 8000;
+const REGTEST_MAKER_IP: &str = "127.0.0.1";
+const REGTEST_MAKER_PORT_HTTP: u64 = 8000;
 // Maker PK is derived from our checked in regtest maker seed
-static REGTEST_MAKER_PK: &str =
-    "02cb6517193c466de0688b8b0386dbfb39d96c3844525c1315d44bd8e108c08bc1";
+const REGTEST_MAKER_PK: &str = "02cb6517193c466de0688b8b0386dbfb39d96c3844525c1315d44bd8e108c08bc1";
 
-static MAKER_PORT_LIGHTNING: u64 = 9045;
+const MAKER_PORT_LIGHTNING: u64 = 9045;
 
-static TESTNET_MAKER_IP: &str = "35.189.57.114"; // testnet.itchysats.network
-static TESTNET_MAKER_PORT_HTTP: u64 = 8888;
-// Maker PK logged in tentenone-maker testnet container
-static TESTNET_MAKER_PK: &str =
+/// IP corresponding to the domain `testnet.itchysats.network`.
+///
+/// _All_ 10101 testnet makers can be found behind this IP.
+const TESTNET_MAKER_IP: &str = "35.189.57.114";
+
+/// The port for the 10101 testnet maker running a stable release.
+///
+/// Alpha users should use this port.
+const TESTNET_STABLE_MAKER_PORT_HTTP: u64 = 8888;
+
+/// The port for the 10101 testnet maker tracking `main`.
+///
+/// Devs should use this port.
+const TESTNET_MAIN_MAKER_PORT_HTTP: u64 = 8889;
+
+/// Public key of the 10101 maker running a stable release.
+///
+/// Alpha users should use this PK.
+const TESTNET_STABLE_MAKER_PK: &str =
     "0244946473b7926c427be70925e8e99cafc3ea76dffe708e6cba8896576cf0b14d";
 
-pub static TCP_TIMEOUT: Duration = Duration::from_secs(10);
+/// Public key of the 10101 maker tracking `main`.
+///
+/// Devs should use this PK.
+const TESTNET_MAIN_MAKER_PK: &str = "unknown";
```

Comment:
Waiting for https://github.com/itchysats/cloud-conf/pull/161 to be reviewed and deployed!